app/src/main/java/org/xwiki/android/sync/contactdb/BatchOperation.java

```diff
@@ -34,49 +34,80 @@
 
 /**
  * This class handles execution of batch mOperations on Contacts provider.
+ *
+ * @version 4.2
```

Comment:
Wrong syntax and this version does not exist.

Reply:
fixed